### PR TITLE
Add in the following snippets

### PR DESCRIPTION
**Pull request by unpluggedk**
- Accept calendar event
- Decline calendar event
- Tentatively accept calendar event
- Get calendar view

### CODE REVIEW

File: Sample/O365-iOS-Snippets/MasterViewController.m [hunk 1]
```diff
@@ -481,6 +486,261 @@ - (void)performDeleteCalendarEvent
                                 }];
 }
 
+- (void) performAcceptCalendarEvent{
```
Comment (MIchaelMainer) [inline Sample/O365-iOS-Snippets/MasterViewController.m]: General comment: we should start summarizing what these methods do. There is setup that these methods perform that may be of interest to reader. Inside each of these methods, we should comment where a method call results in a request to the service.

File: Sample/O365-iOS-Snippets/Office365Snippets.m [hunk 2]
```diff
@@ -425,6 +423,92 @@ - (void)deleteCalendarEvent:(MSOutlookEvent *)event
...
+        NSURLSessionTask *task = [operations tentativelyAcceptWithComment:comment callback:^(int returnValue, MSODataException *error) {
+            BOOL success = (returnValue == 0);
+            completion(success, error);
+        }];
+        [task resume];
+    }];
+}
+
+// Fetches calendar view in specified date range
+// For more information about calendar view, visit https://msdn.microsoft.com/office/office365/APi/calendar-rest-operations#GetCalendarView
```
Comment (MIchaelMainer) [inline Sample/O365-iOS-Snippets/Office365Snippets.m:502]: General comment: Nice! We need to link more often documentation. We should consider this throughout our snippets.

Comment (MIchaelMainer) [inline Sample/O365-iOS-Snippets/Office365Snippets.m:502]: The comments should state that this returns the first 10 event instances.

Comment (unpluggedk) [inline Sample/O365-iOS-Snippets/Office365Snippets.m:502]: Got it, and added.

**Pull request merged**